### ISSUE TITLE
asyncio as dependency?

### ISSUE THREAD

**Issue by MementoRC** [InjectiveLabs/sdk-python#254]
Is it possible to remove asyncio as a dependency? It is only relevant for python <=3.3, but the python support is ^3.9

**Comment by MementoRC** [comment]
@achilleas-kal This is from the PYPI asyncio package: https://pypi.org/project/asyncio/

```
Project description
This version is only relevant for Python 3.3, which does not include asyncio in its stdlib.
```
Your project is python ^3.9

This is bogging down the creation of the conda-forge recipe
Thx

**Comment by MementoRC** [comment]
FYI:
```
import: 'pyinjective'
+ pip check
injective-py 0.9.12 requires asyncio, which is not installed.
WARNING: Tests failed for injective-py-0.9.12-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: injective-py-0.9.12-pyhd8ed1ab_0.conda
```

**Comment by aarmoa** [comment]
This issue is fixed in #301
